lib/diagnostic.js

```diff
@@ -38,18 +51,25 @@ let owner = null;
 // Methods: vocalize
 
 // Replace null with your answer
-let pet = null;
+let pet = [
+  {
+    name: 'Stinky',
+    species: 'cat',
+    breed: 'unknown',
+    vocolize: 'unknown'
+  }
+];
 
 // 4. Create a constructor function for pet objects, as described above. The
 // constructor function should be responsible for setting the values of `name`,
 // `species`, `breed`, and `vocalization`.
 
 const Pet = function() {
-  /* Your answer here */
+  /*  */
 };
 
 // 5. Attach a `vocalize` method to the Pet's prototype.
-
+var vocalize = function()
 /* your answer here */
```

Comment:
I think you're looking for something like this

Pet.prototype.vocalize = function(){
    return this.vocalization;
};